pvlib/tests/iotools/test_psm3.py

```diff
@@ -49,48 +61,72 @@ def assert_psm3_equal(header, data, expected):
     assert (data.index.tzinfo.zone == 'Etc/GMT%+d' % -header['Time Zone'])
 
 
+def delay_get_psm3(*args, **kwargs):
```

Comment:
how about [`pytest-rerunfailures`](https://github.com/pytest-dev/pytest-rerunfailures) instead of this custom approach?

Reply:
I'll look into it.

Reply:
Thanks for the tip. I've implemented pytest-rerunfailures and it works nicely.